torch/nn/utils/rnn.py

```diff
@@ -148,6 +148,11 @@ def _symbolic_pack_padded_sequence(g, input, lengths, batch_first=False, padding
     # optimization pass to remove this later. It is an error if all
     # PackPadded operators cannot be optimized out.
 
+    if not isinstance(input, torch._C.Value):
+        raise RuntimeError("PackPadded requires `input` to be a Tensor")
+    if not isinstance(lengths, torch._C.Value):
+        raise RuntimeError("PackPadded requires `lengths` to be a Tensor")
```

Comment:
Values don't necessarily mean that sth is a tensor. It only means it's not a constant we know statically.

Reply:
Pretty sure we don't support PackPadded in script yet, so the only way we're going to hit this codepath is if we successfully trace an instance of PackPadded, which accepts either a list or a tensor for `value` (and Tensor only for `input`). Can you think of other situations we would hit? This might change in the future, though